roles/network_plugin/calico/handlers/main.yml

```diff
@@ -7,8 +7,13 @@
 
 - name: delete 10-calico.conflist
   file:
-    path: /etc/calico/10-calico.conflist
+    path: /etc/cni/net.d/10-calico.conflist
     state: absent
 
-- name: delete calico-node containers
+- name: docker | delete calico-node containers
```

Comment:
The task reset_calico_cni will fail. You should rewrite it as:

```
---
- name: reset_calico_cni
  command: /bin/true
  notify:
    - delete 10-calico.conflist
    - docker | delete calico-node containers
    - containerd | delete calico-node containers
```

Reply:
thanks, fixed